### ISSUE TITLE
[Thermal Dynamics] Suggestion

### ISSUE THREAD

**Issue by Shinisuryu** [CoFH/Feedback#147]
Not sure if it's in the plans, but would it be possible to have plugs or a variation of the cover that blocks existing ducts from connecting to newly placed ducts? I was testing some ideas and things didn't work very well when the ducts would auto connect to the others and mess up the fluids. Disconnecting ducts after works fine, but then there's the problem of the wrong fluid in the duct meant for a different fluid.

**Comment by Cronosus** [comment]
my suggestion:
please make ducsts again compatible with ForgeMultipart :)

**Comment by Shinisuryu** [comment]
From everything I've heard, that's not gonna happen Cronosus.

**Comment by KingLemming** [comment]
I think a specific cover variant is unlikely but there's probably a solution here somewhere.

And yeah, FMP is staying gone for good. It was a massive headache.

**Comment by Shinisuryu** [comment]
Yeah, I just was thinking off the top of my head. I've noticed filters work, but look sorta weird and out of place. So maybe something based on that but using structural ducts?

Just thinking out loud, haha.

**Comment by KingLemming** [comment]
Well really the issue is that you want to place a duct and not have it auto-join to other ducts, right? A permanent attachment that you only use during placement seems off. I'm thinking some sort of sneak modifier.

**Comment by Shinisuryu** [comment]
Oh right, duh. So you mean that when sneak placing a duct, it won't connect to others? That sounds like it would work since it could then be connected to nearby ducts with a wrench, right?

**Comment by KingLemming** [comment]
Exactly.

**Comment by Shinisuryu** [comment]
That sounds perfect!
